src/components/intern-app.html

```diff
@@ -3,135 +3,130 @@
-                { id: "008", firstName: 'Tony', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "009", firstName: 'Ryan', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "010", firstName: 'Mack', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" }
-              ];
-            }
-          },
-          addUserSelected: {
-            type: Boolean,
-            value: false
-          }
-        }
-      }
+     static get properties() {
+       return {
+         userProfiles: {
+           type: Array,
+           value: () => {
+             return localStorageApi.getUsers(); 
+           }
+         },
+         addUserSelected: {
+           type: Boolean,
+           value: false
+         }
+       }
+     }
 
-      constructor() {
-        super();
-        this.boundNewUser = this.newUser.bind(this);
-      }
+     constructor() {
+       super();
+       this.boundNewUser = this.newUser.bind(this);
+     }
 
-      connectedCallback() {
-        super.connectedCallback();
-        window.addEventListener("newUser", this.boundNewUser);   
-      }
+     connectedCallback() {
+       super.connectedCallback();
+       // localStorageApi.getUsers();
+      //  console.log(localStorageApi.getUser('034'));
+       // localStorageApi.updateUser('352');
+      //  localStorageApi.deleteUser('034');
+       // localStorageApi.deleteUsers();
+       window.addEventListener("newUser", this.boundNewUser);
+     }
 
-      disconnectedCallback() {
-        super.disconnectedCallback();
-        window.removeEventListener("newUser", this.boundNewUser); 
-      }
+     disconnectedCallback() {
+       super.disconnectedCallback();
+       window.removeEventListener("newUser", this.boundNewUser);
+     }
 
-      newUser(event) {
-        var newUserProfile = Object.assign({},event.detail);
-        newUserProfile.id = this.genId();
-        this.pushToArray(this.userProfiles, newUserProfile);
-        this.addUserSelected = false;
-      }
+     newUser(event) {
+       var newUserProfile = Object.assign({},event.detail);
+       newUserProfile.id = this.genId();
+       this.pushToArray(this.userProfiles, newUserProfile);
+       localStorageApi.createUser(newUserProfile);
+       this.addUserSelected = false;
+     }
 
-      pushToArray(userProfiles, newUserProfile) {
-        var existingIds = userProfiles.map((newUserProfile) => newUserProfile.id);
-        if (!existingIds.includes(newUserProfile.id)) {
-          this.push('userProfiles', newUserProfile);
-        }
-        else {
-          newUserProfile.id = this.genId();
-          this.push('userProfiles', newUserProfile);
-        };
-      };
+     pushToArray(userProfiles, newUserProfile) {
+       var existingIds = userProfiles.map((newUserProfile) => newUserProfile.id); 
```

Comment:
can be simplified syntactically
 ```suggestion
       var existingIds = userProfiles.map(newUserProfile => newUserProfile.id); 
```